### PR TITLE
Fix a few SonarCloud code smells in Mixer

### PR DESCRIPTION
**Pull request by oleg-derevenetz**

### CODE REVIEW

File: src/engine/audio_mixer.h [hunk 1]
```diff
@@ -31,31 +31,25 @@
 
 namespace Mixer
 {
-    typedef Mix_Chunk chunk_t;
```
Comment (ihhub) [inline src/engine/audio_mixer.h:34]: Shall we then move `#include <SDL_mixer.h>` to the source file?

**Review by ihhub — COMMENTED** [review]
Hi @oleg-derevenetz , I left just one comment. Would you mind to check?

File: src/engine/audio_mixer.h (hunk 1 above)
Comment (oleg-derevenetz) [reply src/engine/audio_mixer.h:34]: Yes, along with <vector> BTW. I forgot about it.

**Comment by ihhub** [comment]
@oleg-derevenetz , great thanks for cleaning up the code!

**Pull request merged**